src/patternfly/components/Pagination/pagination.scss

```diff
@@ -20,7 +20,9 @@
   --pf-c-pagination__nav-page-select--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-pagination__nav-page-select--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-pagination__nav-page-select--child--MarginRight: var(--pf-global--spacer--xs);
-  --pf-c-pagination__nav-page-select--c-form-control--Width: #{pf-size-prem(48px)}; // custom value to show 2 numbers
+  --pf-c-pagination__nav-page-select--c-form-control--width-base: 3ch;
```

Comment:
Testing this in Safari, looks like it needs 3.5ch with `00` as the value. It makes the box a small amount larger than needed in chrome and firefox, but I think that's OK - should be no harm in making the box bigger than needed, but if it's smaller then characters get cut off. 

3ch - cut off in safari
<img width="1991" alt="Screen Shot 2020-02-05 at 10 24 20 AM" src="https://user-images.githubusercontent.com/35148959/73860844-c7a70380-4801-11ea-973a-d50ed3e22403.png">


3.5ch
<img width="1991" alt="Screen Shot 2020-02-05 at 10 24 07 AM" src="https://user-images.githubusercontent.com/35148959/73860855-cd044e00-4801-11ea-9908-db635eece78c.png">

Reply:
No problem :+1: